src/app/routes/enrolment/enrolment-list/enrolment-list.component.ts

```diff
@@ -231,11 +234,12 @@ export class EnrolmentListComponent implements OnInit, OnDestroy {
         disableClose: true,
       })
       .afterClosed()
-      .pipe(takeUntil(this._subscription$))
-      .subscribe((reloadList: any) => {
-        if (reloadList) {
-          this.getList(this.dynamicRejected, this.dynamicAccepted);
-        }
+      .pipe(
+        truthy(),
+        takeUntil(this._subscription$)
+      )
+      .subscribe(() => {
```

Comment:
All subscriptions should be unsubscribed in ngOnDestroy.
As a best practice I would do:

intitialize a variable at the top `subscriptions = new Subscription();`
then wrap each subscription like this `this.subscriptions.add(.....subscribe()..)`
then in ngOnDestroy just do `this.subscriptions.unsubcribe()`

Reply:
I just saw you are using takeUntil, that can work as well. But you can take this as an idea.

Reply:
Yeah, both solutions are equivalent. I just prefer and i used to use takeUntil. The only thing that there is needed to have in mind, that takeUntil needs to be last operator otherwise there might be unpredicted behavior.